browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -166,14 +176,13 @@ void AIChatUIPageHandler::OpenBraveLeoSettings() {
                                  ui::PAGE_TRANSITION_LINK, false});
 }
 
-void AIChatUIPageHandler::OpenBraveLeoWiki() {
+void AIChatUIPageHandler::OpenURL(const GURL& url) {
   auto* contents_to_navigate = (active_chat_tab_helper_)
                                    ? active_chat_tab_helper_->web_contents()
                                    : web_contents();
-  contents_to_navigate->OpenURL(
-      {GURL("https://github.com/brave/brave-browser/wiki/Brave-Leo"),
-       content::Referrer(), WindowOpenDisposition::NEW_FOREGROUND_TAB,
-       ui::PAGE_TRANSITION_LINK, false});
+  contents_to_navigate->OpenURL({url, content::Referrer(),
```

Comment:
It could be an internal page and that would be ok (if there was one shared between android and desktop, like the eventual `chrome://chat` for full-page conversation)

Reply:
It would be better to decide on an allowlist. If it is `chrome://`, and `https://` it might be fine. At the current stage we are accepting `javascripts://` URLs, which opens up for scriptlets as well.

Reply:
But what is the security concern here? This is our WebUI calling a function in our browser code. If our WebUI is compromised (perhaps via a node_module or similar?), then that code could navigate to any URL or run any javascript and wouldn't need to use this specific function.

Reply:
In the interests of time, we'll do this anyway @nullhook

Reply:
https://github.com/brave/brave-core/pull/20419